DataFormats/TrackReco/interface/TrackBase.h

```diff
@@ -407,14 +424,24 @@ class TrackBase
     /// perigee 5x5 covariance matrix
     float covariance_[covarianceSize];
 
+    /// errors for time and velocity (separate from cov for now)
+    float covt0t0_, covbetabeta_;
+
     /// chi-squared
     float chi2_;
 
     /// innermost (reference) point on track
     Point vertex_;
 
+    /// time at the reference point on track
+    float t0_;
+
     /// momentum vector at innermost point
     Vector momentum_;
+    
+    /// norm of the particle velocity at innermost point on track
+    /// can multiply by momentum_.Unit() to get velocity vector
+    float beta_;
```

Comment:
please define default values here or in the default constructor.
I can't recall now if they also should be specified in the ioread rules (it's easy to check by reading an old file and checking what's in covt0t0_ )